modules/aws/cert-manager.tf

```diff
@@ -137,10 +137,6 @@ resource "helm_release" "cert-manager" {
     local.cert-manager["extra_values"]
   ]
   namespace = kubernetes_namespace.cert-manager.*.metadata.0.name[count.index]
-
-  depends_on = [
-    helm_release.kube-prometheus-stack
```

Comment:
Here I think we should put `kubectl_manifest.prometheus-operator_crds`

Reply:
done